recipes/dooit/recipe.yaml

```diff
@@ -0,0 +1,61 @@
+context:
+  name: dooit
+  version: "3.1.0"
+  python_min: "3.9"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name }}-${{ version }}.tar.gz
+  sha256: ceb0faa4d5a93976a3895505b66c11a80a16c616d6a7d20bbfa9e6c2dbe7c05a
+
+build:
+  noarch: python
+  script: python -m pip install . -vv
+  number: 0
+  python:
+    entry_points:
+      - dooit = dooit.__main__:main
+
+requirements:
+  host:
+    - python ${{ python_min }}
+    - pip
+    - poetry-core
+  run:
+    - python >= ${{ python_min }}
+    - pyyaml >=6.0.2,<6.1
+    - tzlocal >=5.2,<5.3
+    - textual >=0.87.1,<0.88
+    - python-dateutil >=2.9.0.post0,<2.10
+    - sqlalchemy >=2.0.36,<2.1
+    - platformdirs >=4.3.6,<4.4
+    - click >=8.1.7,<8.2
+    - pyperclip >=1.9.0,<2.0
+
+
+tests:
+  - python:
+      imports:
+      - dooit
+      pip_check: true
```

Comment:
Under `pip check`, can you add `python_version: ${{ python_min}}`. I think it would also be good to add a python version under the script:

```
- script:
   - dooit --help
  requirements:
    run:
      - python ${{ python_min }}
 ```